config/parameters.php

```diff
@@ -29,4 +30,9 @@
 
     // cache
     $parameters->set(Option::CACHE_DIR, sys_get_temp_dir() . '/rector_cached_files');
+
+    $runsInGithubAction = getenv('GITHUB_ACTION');
+    if (false !== $runsInGithubAction) {
+        $parameters->set(Option::CACHE_CLASS, MemoryCacheStorage::class);
+    }
```

Comment:
Could you add some comment in here? It might be a bit unclear use for the future readers